examples/clone/clone.bal

```diff
@@ -0,0 +1,69 @@
+import ballerina/io;
+
+//Define anydata record
+public type Person record {
+    string name;
+    int age;
+    boolean married;
+    float salary;
+    Address address;
+};
+
+public type Address record {
+    string country;
+    string state;
+    string city;
+    string street;
+};
+
+// The returned tuple will be later used by ballerina tests
+public function main() returns (Person, Person, string)? {
+    // Create a value of Address
```

Comment:
Shall we say something like "Specify values for the Address record."?

Reply:
Fixed.